src/worker/sandbox/safeSandbox.go

```diff
@@ -118,7 +118,7 @@ func (sb *safeSandbox) DestroyIfPaused(reason string) {
 }
 
 func (sb *safeSandbox) Pause() (err error) {
-	sb.printf("Pause()")
+    //sb.printf("Pause()")
```

Comment:
let's leave these in, because the config already lets you disable all cg logging.